### ISSUE TITLE
At reflow resolution data present in the table is truncating and not visible clearly:A11y_.NetAspireDashboard_Resources_Grid_Reflow

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3317]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

### Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #A11yWCAG2.1; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.4.10; #Reflow;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Prerequisites:

1. Open the web page / screen of the web application that is to be tested
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I"
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as devtools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.

**Alternatively:**

1. Change the OS display settings to a resolution of 1280x1024 at 100% dpi scaling.
2. Open the web page / screen of the web application that is to be tested.
3. Adjust the browser window’s zoom level to 400%.
4. Set the browser to **full screen**. Keyboard shortcut "F11" key.
5. If any browser sidebars are still visible (e.g., the Edge discover sidebar), these _must_ be closed.

### Repro Steps:
1. Hit the URL: http://localhost:15888/.
2. TAB to "Resources" tab item and hit select the tab by pressing Enter key.
3. Observe that at reflow resolution data present in the table is truncating and not visible clearly.

### Actual Results:
At reflow resolution data present in the table is truncating and not visible clearly.

**Similar issue is observed in throughout the application where table is present.**

### Expected Results:
At reflow resolution data present in the table should not be truncating and should be visible clearly.

### User Impact:
Low vision users will not be able to see the data present in the table clearly and will miss the information about it.

### Attachment:
![3317A11y_ NetAspireDashboard_Resources_Grid_AfterReflow](https://github.com/dotnet/aspire/assets/90901519/a3d1f96e-7e08-4395-a044-ef1fae63da96)
![3317A11y_ NetAspireDashboard_Resources_Grid_BeforeReflow](https://github.com/dotnet/aspire/assets/90901519/59730b23-19fd-40bd-a1c4-9e4b9bd680d5)

**Comment by adamint** [comment]
@PoojaNamde can this be closed as duplicate of #3176

**Comment by PoojaNamde** [comment]
@adamint Both the issues are filed for different WCAG standard. This bug is logged for WCAG1.4.10 and https://github.com/dotnet/aspire/issues/3176 is logged for WCAG1.4.4. So this cannot be closed.
